[PG Wrapper] Add sequence number

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #98470
* #97702
* #97692
* #97544
* __->__ #97462

Adds sequence number to PG wrapper

Differential Revision: [D44347419](https://our.internmc.facebook.com/intern/diff/D44347419/)

rohan-varma committed:
[PG Wrapper] Add sequence number

Adds sequence number to PG wrapper

Differential Revision: [D44347419](https://our.internmc.facebook.com/intern/diff/D44347419/)

[ghstack-poisoned]

zhaojuanmao reviewed (approved):
nice, this will be super helpful!

rohan-varma committed:
Update on "[PG Wrapper] Add sequence number"


Adds sequence number to PG wrapper

Differential Revision: [D44347419](https://our.internmc.facebook.com/intern/diff/D44347419/)

[ghstack-poisoned]

rohan-varma committed:
Update on "[PG Wrapper] Add sequence number"


Adds sequence number to PG wrapper

Differential Revision: [D44347419](https://our.internmc.facebook.com/intern/diff/D44347419/)

[ghstack-poisoned]

rohan-varma committed:
Update on "[PG Wrapper] Add sequence number"


Adds sequence number to PG wrapper

Differential Revision: [D44347419](https://our.internmc.facebook.com/intern/diff/D44347419/)

[ghstack-poisoned]
